README.md

```diff
@@ -57,10 +65,18 @@ avroMessageIdToDictionaryAlias:
   '1': "${dictionary_link:avro-schema-1-dictionary}"
   '2': "${dictionary_link:avro-schema-2-dictionary}"
   '3': "${dictionary_link:avro-schema-3-dictionary}"
+sessionAliasToDictionaryAlias:
+  'sessionAlias1': "${dictionary_link:avro-schema-1-dictionary}"
+  'sessionAlias2': "${dictionary_link:avro-schema-2-dictionary}"
+  'sessionAlias3': "${dictionary_link:avro-schema-3-dictionary}"
```

Comment:
I think we need wildcard support. You can use the same example as common
https://github.com/th2-net/th2-common-j/blob/74e55f2e064f7ec33987012dc9624bd1e14322d3/src/main/kotlin/com/exactpro/th2/common/schema/filter/strategy/impl/FieldValueChecker.kt#L29